kernel/src/main.rs

```diff
@@ -68,12 +69,16 @@ extern "C" fn main(mem_upper: usize, video_memory_skip_lines: usize) -> ! {
 
         println!("Setting up block layer");
         let block_manager = block_init();
-        println!("{}", block_manager);
         println!("Block layer set up!");
 
         println!("Initializing Thread System...");
         thread_system_initialization();
         println!("Finished Thread System initialization. Ready to start threading.");
+
+        println!("Setting up IDE");
+        let _block_manager = ide_init(block_manager, true);
```

Comment:
```suggestion
        let block_manager = ide_init(block_manager, true);
```

I mean, `block_manager` is moved here anyway.

Reply:
Oh this is for pacifying the linter so it does not block the commit.
```
   = note: `-D unused-variables` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(unused_variables)]`
```